latest/index.bs

```diff
@@ -175,7 +175,261 @@ For this example we assume an image with 5 dimensions and axes called `t,c,z,y,x
                 └── n
 </pre>
 
+Tables {#table-layout}
+----------------------
+The following describes the expected layout for tabular data. OME-NGFF tables are compatible with the [AnnData model](https://github.com/theislab/anndata).
 
+<pre>
+.                             # Root folder, potentially in S3,
+│                             # with a flat list of images by image ID.
+│
+└── 123.zarr
+    └── table                 # The table group is a container which holds a table that is compatible with AnnData.
+        │                     # The table group MAY be in the root of the zarr file.
+        ├── .zgroup           # The table group MAY be in an image, labels, or points group.
+        |
+        ├── .zattrs           # `.zattrs` MUST contain "@type", which is set to `"ngff:table"`
+        |                     # `.zattrs` MAY contain "parent", which is the relative path to the data the table is annotating.
```

Comment:
Is `"parent"` the best name for this key? Also, what do we think about using relative path to link to the image/labels/points/ROIs?

cc @giovp ,  @ivirshup , @aeisenbarth

Reply:
> Why not region?

Ah right. I somehow forgot about that 🤦 

> Relative to what? If it's relative to the table, I'm not sure this will work as zarr doesn't support ..

I had been thinking relative to the table. While `..` isn't supported by zarr, the reader could construct the path. I'm not sure it's the best path forward though.

Reply:
I've updated to include "region", "region_key", and "instance_key". Given the proposed default behavior for "instance_key", I think `obs` is now required.

Reply:
> I had been thinking relative to the table. While .. isn't supported by zarr, the reader could construct the path. I'm not sure it's the best path forward though.

I personally would like to avoid defining a specification for path strings. Seems like there are a lot of edge cases.


-----------


I think the path issue can be sidestepped if we move to the flat structure we proposed. Here, you could have

```
tables/table_0/.zattrs {..., "region": "region_0"}
```

which would refer to

```
regions/region_0
```

Reply:
It looks like `image_labels` currently expects the reader to implement `".."` handling, as the default value for `"image"` is currently `"../../"`.

Reply:
As general feedback, I'm a bit skeptical about a fully flat structure and would prefer us to have a generally re-usable path handling whether it be support for relatives including `".."` or just absolutes everywhere.

Reply:
`ome-zarr-py` handles the relative paths itself using the [`create`](https://github.com/ome/ome-zarr-py/blob/313e12690fd95d4f199ef3aba9d9412730012c53/ome_zarr/io.py#L134-L138) method [(called here)](https://github.com/ome/ome-zarr-py/blob/313e12690fd95d4f199ef3aba9d9412730012c53/ome_zarr/reader.py#L218).

I'd be fine doing the same thing for now, and pushing paths to a separate PR. I think it would make sense to have path handling be its own discussion.